documentation/bolt_project_directories.md

```diff
@@ -108,6 +108,27 @@ Bolt uses these methods, in order, to choose a project directory.
 1. **User project directory:** If no directory is specified manually or found in
    a parent directory, the user project directory is used.
 
+## World-writable project directories
+
+By default, Bolt will not load a project from a world-writable directory, as
+they present a potential security risk. If you attempt to load a project from
+a world-writable directory, Bolt will not load any content and will raise an
+exception.
```

Comment:
```suggestion
By default, Bolt will not load a project from a world-writable directory, because
loading from a world-writable directory presents a potential security risk. 
If you attempt to load a project from a world-writable directory, Bolt will not load 
any content and will raise an exception.
```